packages/core/core/src/Parcel.js

```diff
@@ -321,6 +321,10 @@ export default class Parcel {
             };
           }
 
+          // If this bundle was previously a placeholder, it now no longer is as it has
+          // been requested.
+          bundleNode.value.isPlaceholder = false;
```

Comment:
Not sure if this is the right place or way to do this - is it "safe" to mutate a `bundleNode` here?

Reply:
hmm kinda weird but probably ok since we only store the bundle graph in the cache when parcel exits.

Reply:
Actually, I just had another look and I have a feeling this might be required with the `this.safeToIncrementallyBundle = false;` fixes, as they'll cause the bundle graph to be recreated and the placeholder flag will be appropriately set.

My tests seem to pass without this line, I'll just validate it's the `safeToIncrementallyBundle` change that's fixing the issue.

Reply:
Cool, I've removed this change as the other change (which I made later) in `AssetGraph` fixes the issue this was fixing.